src/ui/public/vislib/visualizations/point_series/heatmap_chart.js

```diff
@@ -54,6 +55,7 @@ export function VislibVisualizationsHeatmapChartProvider(Private) {
           labels.push(`${from} - ${to}`);
         });
       } else {
+        if (max === min) max += 1;
```

Comment:
This book-ending with an integer is a little arbitrary. E.g what to do with ranges within decimal ranges?

It also creates an arbitrary legend with "fake" values where there should be none.

![image](https://cloud.githubusercontent.com/assets/1833023/25539025/2c0e352c-2c13-11e7-81bf-202fc4295360.png)

I would consider rewriting this whole label lookup with a special case for where `min===max` and not create a dummy range so we can reuse the same code-path